commands/create_cluster.go

```diff
@@ -23,11 +23,11 @@ type cpuDefinition struct {
 }
 
 type memoryDefinition struct {
-	SizeGB int `yaml:"size_gb,omitempty"`
+	SizeGB float32 `yaml:"size_gb,omitempty"`
 }
 
 type storageDefinition struct {
-	SizeGB int `yaml:"size_gb,omitempty"`
+	SizeGB float32 `yaml:"size_gb,omitempty"`
```

Comment:
Hmm. Not convinced. I expect a maximum of two or three digits after the comma in practice. Why not use the leaner type then?